internal/api/instrumentation/probes.go

```diff
@@ -118,6 +129,11 @@ func RunCreated(ctx context.Context, recipient uuid.UUID, runId uuid.UUID, paylo
 	runCreatedTotal.WithLabelValues(service, requestType).Inc()
 }
 
+func RunCanceled(ctx context.Context, runId uuid.UUID, requestType string) {
+	utils.GetLogFromContext(ctx).Infow("Canceled playbook run", "run_id", runId.String())
```

Comment:
Might be worth clarifying the message - e.g. "successfully initiated playbook run cancelation" to make it obvious that the operation was not finished, only successfully started.